src/commons/assessment/Assessment.tsx

```diff
@@ -349,6 +349,21 @@ const Assessment: React.FC<AssessmentProps> = props => {
     );
   }
 
+  // Define the warning text when finalising submissions
+  const hasBonusXp = (betchaAssessment?.earlySubmissionXp as number) > 0;
+  console.log(hasBonusXp);
```

Comment:
Let's remove this console.log statement

Reply:
sorry, i can't believe i missed that. will fix right away

Reply:
No worries, thanks!